flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/dataformat/DataFormatConverters.java

```diff
@@ -252,6 +253,11 @@ public static DataFormatConverter getConverterForDataType(DataType originDataTyp
 					return BinaryGenericConverter.INSTANCE;
 				}
 				return new GenericConverter(typeInfo.createSerializer(new ExecutionConfig()));
+			case TIMESTAMP_WITHOUT_TIME_ZONE:
```

Comment:
https://github.com/apache/flink/pull/10418

Reply:
Do you mean we should wait for #10418 to get in first?

Reply:
No, you can discuss with @docete